### DISCUSSION TITLE
Error when trying to upgrade to Qodana 2023.2.1

### DISCUSSION THREAD

**Discussion by segevfiner** [JetBrains/Qodana#183] [Q&A]
Trying to update to Qodana (`JetBrains/qodana-action`) 2023.2.1 just gives me this error:
```
Can't find script implementation for 'local-changes' value
```
While the same workflow works fine with 2022.3.4

Any ideas? Feels like something is wrong in that version…

**Comment by brichbash** [comment]
Could you please share the full Qodana output and also the Qodana job configuration snippet?

**Comment by segevfiner** [comment]
Looks like I forgot to also update the linter version in qodana.yaml.
